components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -488,8 +562,8 @@ export default class PostList extends React.PureComponent {
                             onClick={this.scrollToBottom}
                         />
                     </React.Fragment>
-                )}
-                {newMessagesBelow}
+                ) && newMessagesBelow}
```

Comment:
Can you move this into the above Fragment?

Reply:
I'll try again, for some reason i moved into the fragment and it gave me an error, but didn't put too much thinking into it, just moved out of the way. Will change it into the fragment above or next to it.